research-software-directory/RSD-as-a-service
# CodeMeta Docker image size

## CodeMeta Docker image size

Changes proposed in this pull request:

* Use a builder and a runner image for the CodeMeta module, reducing the image size from 840MB to 89MB
* Upgrade Go from 1.22.0 to 1.22.1
* Upgrade a GitHub action to get rid of the last Node 16 deprecation warning when releasing

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Browse the CodeMeta overview (http://localhost/metadata/codemeta), check some of the pages

Related to #1096

PR Checklist:
